Makefile

```diff
@@ -111,22 +114,22 @@ installlib-%: $(JBUILDER_DEP) opam-installer opam-%.install
 	$(if $(wildcard src_ext/lib/*),\
 	  $(error Installing the opam libraries is incompatible with embedding \
 	          the dependencies. Run 'make clean-ext' and try again))
-	$(JBUILDER) exec $(JBUILDER_ARGS) -- opam-installer $(OPAMINSTALLER_FLAGS) opam-$*.install
+	$(OPAMINSTALLER) $(OPAMINSTALLER_FLAGS) opam-$*.install
 
 uninstalllib-%: $(JBUILDER_DEP) opam-installer opam-%.install
```

Comment:
`$(JBUILDER_DEP)` could probably be removed from here (and below) now that the command doesn't use `$(JBUILDER)` anymore.